content/en/docs/tutorials/compose-migration/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Migrating from compose workflows to Kubernetes
+weight: 60
+---
+
+## Disclaimer
+
+The [compose specification](https://github.com/compose-spec/compose-spec) is evolving, this documentation aims to cover the latest version of this compose specification.
+We will assume that we are talking about the latest compose specifications which the v3 at this time of writing.
+If your compose files at not at the latest version specification, we suggest you try to upgrade to the latest spec version first. 
+
+## Differences between compose specification and Kubernetes specification
+
+A set of workflows defined using the compose specification cannot be automated translated to a unique set of Kubernetes objects.
+While some notions are very similar, there is not a single way to translate concepts from compose to Kubernetes.
+There is no direct equivalence for many concepts of compose in Kubernetes.
+
+Also, you have features that are available in the compose specifications that do not have a clear equivalent in Kubernetes.
+
+Therefore, these content are only there to be suggestions and not a definitive and absolute guide.
+
+There is an [ongoing work](https://github.com/compose-spec/compose-spec/blob/wip_kube_mapping/KUBERNETES_MAPPING.md) on the compose specification side to give some information about compose to Kubernetes mapping.
```

Comment:
```suggestion
There is a [Compose - Kubernetes mapping](https://github.com/compose-spec/compose-spec/blob/wip_kube_mapping/KUBERNETES_MAPPING.md) on the compose specification side to give some information about compose to Kubernetes mapping.
```
Perhaps it's better to just list the name of that page than to say "ongoing work", since the page hasn't been updated for about two years.